bot0.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
could you rename the filename to `arbitrage.go`?

Reply:
done